manuscripts/esquery.py

```diff
@@ -29,29 +29,26 @@
 
 from elasticsearch_dsl import A, Search, Q
 
-USE_ELASTIC_DSL = True
-
 
 class ElasticQuery():
     """ Helper class for building Elastic queries """
 
     AGGREGATION_ID = 1  # min aggregation identifier
     AGG_SIZE = 100  # Default max number of buckets
-    ES_PRECISION = 3000
+    ES_PRECISION = 3000  # This is the default value for percision_threshold
 
     @classmethod
     def __get_query_filters(cls, filters=None, inverse=False):
         """
         Convert a dict with the filters to be applied ({"name1":"value1", "name2":"value2"})
-        to a DSL string representing these filters.
+        to a list of match filters which can be used together in a query using boolean combination
+        logic.
 
         :param filters: dict with the filters to be applied
         :param inverse: if True include all the inverse filters (the one starting with *)
-        :return: a string with the DSL value for these filters
+        :return: a list of match object(dict)(s)
```

Comment:
In all references to elasticsearch_dsl objects, I would expliticly mention the module. For example in this case, "list of elasticsearch_dsl match objects", or if you prefer, "es_dsl" for short. Please, add this to all this kind of references in other methods, so that it is crystal clear what they do.

Besides, what do you mean by (dict)(s)? It is better that you describe in detail the data structure that the method returns, including, if it is convenient, an example. Remember that this message should include enough information for a trained developer to be able of using the method.

Reply:
By dicts i meant that the filters are dictionaries. I'll update the comments to be more clear.

Reply:
Thx